lib/livebook/file_system/s3.ex

```diff
@@ -76,8 +76,7 @@ end
 
 defimpl Livebook.FileSystem, for: Livebook.FileSystem.S3 do
   alias Livebook.FileSystem
-  alias Livebook.Utils.HTTP
-  alias Livebook.FileSystem.S3.XML
+  alias Livebook.FileSystem.S3.Client, as: S3Client
```

Comment:
```suggestion
  alias Livebook.FileSystem.S3
```

and just `S3.Client`?